packages/backend/src/index.ts

```diff
@@ -59,6 +59,20 @@ let redisClient: undefined | Awaited<ReturnType<typeof createRedisClient>>;
     };
     app.locals.io = io;
 
+    app.locals.keys = {
+        signing: {
+            publicKey: process.env.SIGNING_PUBLIC_KEY!,
```

Comment:
I added `readKeysFromEnv()` to ensure that the key is not `undefined`.
I wouldn't put keys into the config.yml because most hosting providers and also github actions provide a secure store for keys which are then injected as an env variable